iscsi/get.go

```diff
@@ -112,24 +112,21 @@ func ReadWriteOPS(iqnPath string, tpgt string, lun string) (readmb uint64,
 		"statistics/scsi_tgt_port/read_mbytes")
 	readmb, err = util.ReadUintFromFile(readmbPath)
 	if err != nil {
-		return 0, 0, 0, fmt.Errorf("iscsi: ReadWriteOPS: read_mbytes error file %s and %v",
-			readmbPath, err)
+		return 0, 0, 0, fmt.Errorf("iscsi: ReadWriteOPS: read_mbytes error file %s: %w", readmbPath, err)
 	}
 
 	writembPath := filepath.Join(iqnPath, tpgt, "lun", lun,
 		"statistics/scsi_tgt_port/write_mbytes")
 	writemb, err = util.ReadUintFromFile(writembPath)
 	if err != nil {
-		return 0, 0, 0, fmt.Errorf("iscsi: ReadWriteOPS: write_mbytes error file %s and %v",
-			writembPath, err)
+		return 0, 0, 0, fmt.Errorf("iscsi: ReadWriteOPS: write_mbytes error file %s: %w", writembPath, err)
 	}
 
 	iopsPath := filepath.Join(iqnPath, tpgt, "lun", lun,
 		"statistics/scsi_tgt_port/in_cmds")
 	iops, err = util.ReadUintFromFile(iopsPath)
 	if err != nil {
-		return 0, 0, 0, fmt.Errorf("iscsi: ReadWriteOPS: in_cmds error file %s and %v",
-			iopsPath, err)
+		return 0, 0, 0, fmt.Errorf("iscsi: ReadWriteOPS: in_cmds error file %s: %w", iopsPath, err)
```

Comment:
Here (and similarly above), I think, a `%q` would actually make more sense than a `%s` because it is in the middle of the message, not after the final `:`. WDYT?

Reply:
Yes, that is one we should cleanup. Will add it.